Assets/Tests/Samples/InGameHintsTests.cs

```diff
@@ -15,6 +15,7 @@ public class InGameHintsTests : CoreTestsFixture
 {
     [UnityTest]
     [Category("Samples")]
+    [Ignore("Disabled: Unstable and running into Debug.Assert in ~@InGameHintsActions() on rare occasions.")]
```

Comment:
I am not familiar with this syntax. Is this to ignore test failure?

Reply:
the whole test is going to get ignored aka not run. is there a better way? never did this before myself.

Reply:
Interesting, me neither. So unfortunately I don't know.

Reply:
@stefanunity this is the way :)

Reply:
@stefanunity I would also recommend filing a ticket on any unstable tests disabled in this PR when its merged.

Reply:
https://jira.unity3d.com/browse/ISX-1896 for followup